docs/workforce-management-framework/topics/con-introducing-securityfeatures.adoc

```diff
@@ -2,11 +2,11 @@
 = Introducing {WFM-RC-NameShort} Security Features
 
 {WFM-RC-NameShort} Security is built around a *Security Interface* which abstracts itself from all {WFM-RC-NameShort} security libraries.
-{WFM-RC-NameShort} allows you to build a custom security solution for integration with {WFM-RC-NameShort}.
-{WFM-RC-NameShort} also provides two security implementations out-of-the-box and are called link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} allows you to integrate custom security solution.
+{WFM-RC-NameShort} also provides two out-of-the-box security implementations: link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
```

Comment:
+1 for shortening the text. Best to drop the " : " and use English where possible, " security implementations called ..."

Reply:
will do